applications/dashboard/views/role/edit.php

```diff
@@ -1,10 +1,4 @@
-<?php if (!defined('APPLICATION')) exit(); ?>
```

Comment:
@R-J You need it in any file that prints something directly to the output. Files that don't (ala plugin files, models, etc.), don't need it.

The idea is to prevent output from happening unless it is through the framework (i.e. APPLICATION is defined).